sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicOperators.scala

```diff
@@ -134,6 +134,16 @@ case class CreateTableAsSelect[T](
   override lazy val resolved = databaseName != None && childrenResolved
 }
 
+/**
+ * @param child The final query of this CTE
```

Comment:
`A container for holding named common table expressions (CTEs) and a query plan.  This operator will be removed during analysis and the relations will be substituted into `child`.`